src/geff/_cli.py

```diff
@@ -1,31 +1,159 @@
+        ),
+    ] = None,
+    input_image_dir: Annotated[
+        Path | None,
+        typer.Option(
+            help="The path to the input image directory. If not provided, it won't be exported."
+        ),
+    ] = None,
+    output_image_path: Annotated[
+        Path | None,
+        typer.Option(
+            help="The path to export the image file, if not provided, it won't be exported."
+        ),
+    ] = None,
+    tczyx: Annotated[
+        bool,
+        typer.Option(
+            help="Expand data to make it (T, C, Z, Y, X) otherwise it's (T,) + Frame shape."
+        ),
+    ] = False,
+    overwrite: Annotated[
+        bool, typer.Option(help="Whether to overwrite the GEFF file if it already exists.")
+    ] = False,
+) -> None:
+    """
+    Convert a CTC data directory to a GEFF file.
+    """
+    from geff.interops import from_ctc_to_geff, ctc_tiffs_to_zarr  # noqa: I001 import at call time to avoid optional dependency issues
+
+    if (input_image_dir is not None and output_image_path is None) or (
+        input_image_dir is None and output_image_path is not None
+    ):
+        raise ValueError("'input_image_dir' and 'output_image_path' must be provided together")
+
+    from_ctc_to_geff(
+        ctc_path=ctc_path,
+        geff_path=geff_path,
+        segmentation_store=segm_path,
+        tczyx=tczyx,
+        overwrite=overwrite,
+    )
+
+    if input_image_dir is not None:
+        ctc_tiffs_to_zarr(
+            ctc_path=input_image_dir,
+            output_store=output_image_path,
+            ctzyx=tczyx,
+            overwrite=overwrite,
+        )
+
+
+@app.command()
+def convert_trackmate_xml(
+    xml_path: Annotated[
+        Path, typer.Argument(help="The path to the TrackMate XML file", show_default=False)
+    ],
+    geff_path: Annotated[
+        Path,
+        typer.Argument(
+            help="Path to save the output geff, including the geff directory name"
+            " (eg. ~/folder/folder/data.zarr/tracks.geff)",
+            show_default=False,
+        ),
+    ],
+    discard_filtered_spots: Annotated[
+        bool,
+        typer.Option(help="True to discard the spots filtered out in TrackMate, False otherwise."),
+    ] = False,
+    discard_filtered_tracks: Annotated[
+        bool,
+        typer.Option(help="True to discard the tracks filtered out in TrackMate, False otherwise."),
+    ] = False,
+    overwrite: Annotated[
+        bool, typer.Option(help="Whether to overwrite the GEFF file if it already exists.")
+    ] = False,
+    zarr_format: Annotated[int, typer.Option(help="The version of zarr to write.")] = 2,  # type: ignore
+    # because of Typer not supporting Literal types
+) -> None:
+    """
+    Convert a TrackMate XML file to a GEFF file.
+    """
+    from geff.interops import from_trackmate_xml_to_geff  # noqa: I001 import at call time to avoid optional dependency issues
```

Comment:
```suggestion
    from geff.interops import from_trackmate_xml_to_geff 
```

(unneeded noqa ... ruff doesn't mind imports inside functions, they're fine... and absolutely standard in this scenario)